tests/foreman/ui/test_host.py

```diff
@@ -1695,6 +1695,8 @@ def test_global_registration_upgrade_subscription_manager(
         4. Check subscription-manager was installed from repo_name
 
     :parametrized: yes
+
+    :BZ: 1923320
```

Comment:
also `customerscenario: true`

Reply:
Yes the Customer Cases are attached to the BZ so its worth adding customerscenario testimony token.